android/app/src/main/java/app/organicmaps/sdk/util/log/LogsManager.java

```diff
@@ -114,9 +115,9 @@ private String ensureLogsFolder()
     if (mLogsFolder != null && createWritableDir(mLogsFolder))
       return mLogsFolder;
 
-    mLogsFolder = createLogsFolder(mApplication.getExternalFilesDir(null));
+    mLogsFolder = createLogsFolder(Objects.requireNonNull(mApplicationContext).getExternalFilesDir(null));
```

Comment:
Is it worth calling requireNonNull(X) here and below when just calling a function?
Syntax overhead, IMO.